Homework/Develop/assets/css/style.css

```diff
@@ -8,39 +8,43 @@ body {
     background-color: #d9dcd6;
 }
 
-.header {
+
+
+header {
     padding: 20px;
     font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
     background-color: #2a607c;
     color: #ffffff;
 }
 
-.header h1 {
+header h1 {
     display: inline-block;
     font-size: 48px;
 }
 
-.header h1 .seo {
+header h1 .seo {
     color: #d9dcd6;
 }
 
-.header div {
+Nav {
```

Comment:
Please check this to make sure you have the right case

Reply:
updated this here thanks 8b32038a16205c6727065cc1712ce5cd6c7a0285